packages/zoe/src/contracts/priceAggregatorChainlink.js

```diff
@@ -1,798 +0,0 @@
-// @ts-check
```

Comment:
I don't recall why we don't need this any more. I wonder if that disqualifies me as a reviwer.

Reply:
I think I see now.